pkg/ovs/openflow/ofctrl_nxfields.go

```diff
@@ -90,6 +90,10 @@ func (m *CtMark) GetValue() uint32 {
 	return m.value
 }
 
+func (m *CtMark) GetValueWithOffset() uint32 {
```

Comment:
Please add comments for this function. Besides, do you think we still need the function `GetValue()`?

Reply:
`GetValue()` is not used by any other functions. I think we can remove it or name this function as `GetValue()`

Reply:
I also think we could just use this method to replace the original `GetValue` method. There should be no need to get the original value without offset from the perspective of usage. For example, even if there is a mark `NewCTMark(0x1, 4, 8)`, the code will never use 0x1 directly for matching. Instead, it will use 0x1<<4/MASK in OpenFlow and value & MASK == 0x1<<4 in Golang code.